Makefile

```diff
@@ -52,65 +99,68 @@ hobby-erc20: $(patsubst %, $(specs_dir)/hobby-erc20/%, $(hobby_erc20_files)) $(s
 
 sum-to-n: $(specs_dir)/examples/sum-to-n-spec.k $(specs_dir)/lemmas.k
 
-
-# Definition Files
-# ----------------
-
-# Lemmas
-$(specs_dir)/lemmas.k: resources/lemmas.md
-	@echo >&2 "== tangle: $@"
-	mkdir -p $(dir $@)
-	pandoc --from markdown --to "$(TANGLER)" --metadata=code:".k" $< > $@
-
-# Spec Files
-# ----------
-
 # Bihu
-$(specs_dir)/bihu/collectToken-spec.k: bihu/module-tmpl.k bihu/spec-tmpl.k bihu/collectToken-spec.ini
+bihu_tmpls:=bihu/module-tmpl.k bihu/spec-tmpl.k
+
+$(specs_dir)/bihu/collectToken-spec.k: bihu/collectToken-spec.ini $(TANGLER)
```

Comment:
why did you drop the tmpls from the dependency list?

Reply:
Well, if they're static, then I think they don't belong in the prereqs. If they're generated, then they do. My impression was that they're static.

The reason I did is is because I wanted to make these targets depend on the TANGLER, but now I realize that they actually don't depend on the TANGLER. I will remove that dependency and force push.